specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/CreateDatabaseConfiguredBackupStorageRedundancy.json

```diff
@@ -0,0 +1,86 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default-SQL-SouthEastAsia",
+    "serverName": "testsvr",
+    "databaseName": "testdb",
+    "api-version": "2020-11-01-preview",
+    "parameters": {
+      "location": "southeastasia",
+      "requestedBackupStorageRedundancy": "Zone"
```

Comment:
The model validation error caused by this property needs to be within `properties` object instead of top level of body.

Reply:
requestedBackupStorageRedundancy is here to show an example of how to _set_ this property when creating a database

Reply:
Thank you so much!